src/Build/BackEnd/Components/RequestBuilder/RequestBuilder.cs

```diff
@@ -1409,7 +1430,7 @@ private void ConfigureKnownImmutableFolders()
             }
         }
 
-        private ISet<string> ParseWarningCodes(string warnings)
+        private static ISet<string> ParseWarningCodes(string warnings)
```

Comment:
nit: is it required for the change? What is the pros?

Reply:
Not necessary, but makes it more understandable that there cannot be any side-effects